### DISCUSSION TITLE
I want to specify which github org my vercel account uses

### DISCUSSION THREAD

**Discussion by ktm** [vercel/community#6002] [Account Issue]
### Question

I am a freelancer and my GH account has several orgs: my personal, my business, my clients.  I want to connect my vercel account to my GH personal or GH business.  When I go to deploy my 1st project (from templates) it flows to 'Create Git Repository' and when I hit the 'Github' button it presents an overlay which appears to want to single sign-on to one of my client's orgs.  I think it obvious that my personal vercel account should be sandboxed to my personal GH orgs.  Ideally one of my own choosing.  Is this not the case?  Do I need to create a different GH ID for my personal vercel account?

**Comment by xonx4l** [comment]
There are a few ways to control which GitHub organization Vercel connects to when deploying projects:

When you first connect Vercel to GitHub, it will default to using your personal account. You can change this in your Vercel account settings under "Git Integrations".

When deploying a new project, Vercel will prompt you to create a Git repository. At this step, you can choose which organization to create the repo under by using the dropdown next to the owner name.

If you've already created the repo on GitHub, you can specify the owner when linking the repo in Vercel.

You can use GitHub Teams to control access to repositories, and connect your Vercel account to the team instead of the organization directly.

So in summary - you don't need separate GitHub accounts. Just be sure to pay attention to which owner/org you are selecting when creating new repos and connecting existing ones. Vercel gives you control over this, it just takes a bit of care.
